client/app/telecom/telephony/line/calls/externalNumber/telecom-telephony-line-calls-externalNumber.html

```diff
@@ -48,7 +48,7 @@ <h1 data-translate="loading"></h1>
                                 <span data-ng-bind="number.number | phoneNumber"></span>
                             </td>
                             <td class="align-middle">
-                                <span class="normal"
+                                <span class="font-weight-normal"
```

Comment:
`td  span` === Hum, it's not bold right ?

So why `font-weight-normal` is needed ?